packages/redux/action-creator-dispatcher/mixin.server.js

```diff
@@ -11,14 +11,16 @@ const ReduxActionCreatorCommonMixin = require('./mixin.runtime-common');
 class ReduxActionCreatorServerMixin extends ReduxActionCreatorCommonMixin {
   bootstrap(request) {
     this.request = request;
-    this.prefetchedOnServer = this.shouldPrefetchOnServer();
+
+    const prefetchOnServer = this.canPrefetchOnServer().every(value => value);
```

Comment:
Can you make this equivalent to `hops-graphql` and execute `this.canPrefetchOnServer` in the `fetchData()` hook?